.claude/skills/feature-planner/SKILL.md

```
@@ -0,0 +1,317 @@
+# Ruby
+bundle exec rspec --coverage
+open coverage/index.html
+
+# PHP
+phpunit --coverage-html coverage
+```
+
+### Common Test Patterns
+
+**Arrange-Act-Assert (AAA) Pattern**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)